slidez-core/src/main/java/com/binarystudio/academy/slidez/link/model/Link.java

```diff
@@ -0,0 +1,37 @@
+package com.binarystudio.academy.slidez.link.model;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import javax.persistence.*;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.GenericGenerator;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Table(name = "link")
+public class Link {
+
+	@Id
+	@GeneratedValue(generator = "UUID")
+	@GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
+	@Column(name = "link_id", updatable = false, nullable = false)
+	private UUID linkId;
```

Comment:
Initially, this field should've been a Long, but since we're using links as an order(which is plausible), this is ok